metabox/metabox/core/scenario.py

```diff
@@ -168,7 +169,7 @@ def start_all(self, interactive=False, timeout=0):
 
     def start_remote(self, interactive=False, timeout=0):
         outcome = self.remote_machine.start_remote(
-            self.service_machine.address, self.LAUNCHER_PATH, interactive,
+            self.service_machine.address, self.LAUNCHER_PATH, cmd_args=self.cmd_args, interactive=interactive,
```

Comment:
Please run `black` to format the code, so we don't get long lines like this.

Reply:
how do you run black on checkbox code ? when i run it, i have at the end a bunch of changes ....

Reply:
Because we have so many PR underway we haven't run black on all of the codebase, so we're trying to reformat only the bits we touch. Preferably before we start editing them, as the first commit.

Reply:
done